README.md

```diff
@@ -158,7 +158,7 @@ Example commands:
 
 Install this project with:
 - cd my-folder
--  git clone https://github.com/Victor-chiemerie/Awesome_books_ES6
+-  git clone https://github.com/Victor-chiemerie/To_do_list
 
 install npx
```

Comment:
- [ ] Kindly check and add instructions that are working to run your project locally.  Get the steps in an orderly manner. This makes it easier for non-expert to run and test your project with ease.